asm/f64/add_amd64.s

```diff
@@ -31,7 +31,7 @@ add_loop:
 	JE	add_end
 add_tail:
 	MOVSD	(DI)(AX*8), X0
-	DIVSD	(SI)(AX*8), X0
```

Comment:
Add started by copying over the div code and I had commit before switching machines.  Bad commit management on my part.